src/components/inputs/money-input/money-input.spec.js

```diff
@@ -327,9 +327,9 @@ describe('MoneyInput', () => {
     expect(getByLabelText('Amount')).toHaveAttribute('data-foo', 'bar');
   });
 
-  it('should render a number input', () => {
+  it('should render a text input', () => {
     const { getByLabelText } = render(<TestComponent />);
-    expect(getByLabelText('Amount')).toHaveAttribute('type', 'number');
+    expect(getByLabelText('Amount')).toHaveAttribute('type', 'text');
```

Comment:
IMHO we can delete this spec. The user doesn't care what kind of input we render as long as they can type. Even if we rendered a content-editable div, they wouldn't care. So I guess this spec can just go away completely.

I wasn't sure whether I should add it back then, and it's kinda annoying that we had to change the tests just because the implementation changed. Users don't care about the type here.